generate_configurations.py

```diff
@@ -28,7 +28,7 @@ def is_pypy(self):
 PYTHON_VERSIONS = ['python2.6', 'python2.7', 'python3.2', 'python3.3',
                    'python3.4', 'python3.5', 'pypy', 'pypy3']
 PYTEST_VERSIONS = ['2.7.3', '2.8.0']
-DJANGO_VERSIONS = ['1.4', '1.5', '1.6', '1.7', '1.8', '1.9', 'master']
+DJANGO_VERSIONS = ['1.4', '1.5', '1.6', '1.7', '1.8', '1.9',]
```

Comment:
Currently (https://travis-ci.org/pytest-dev/pytest-django/builds/82408028) the only tests that runs against Postgres/MySQL is allowed failures (that also runs against master).

We need to have Postgres/MySQL running regularly with 1.4-1.9 and to have those build fail the entire build if there are failures.